Framework/include/QualityControl/TaskDataProcessorFactory.h

```diff
@@ -0,0 +1,29 @@
+///
+/// \file   TaskDataProcessorFactory.h
+/// \author Piotr Konopka
+///
+
+#ifndef PROJECT_TASKDATAPROCESSORFACTORY_H
+#define PROJECT_TASKDATAPROCESSORFACTORY_H
+
+#include "Framework/DataProcessorSpec.h"
+
+namespace o2 {
+namespace quality_control {
+namespace core {
+
+/// \brief Factory in charge of creating DataProcessorSpec of QC task
+class TaskDataProcessorFactory
```

Comment:
I know we discussed it but I am not 100% sure what was the conclusion about the name of this class. `DataProcessorFactory` ?

Reply:
Mmm maybe I am mistaken and it was TaskDataProcessorFactory :)